configure.ac

```diff
@@ -3928,6 +3928,14 @@ if test $GPU_SUPPORT = "CUDA" ; then
     fi
 fi
 
+if test $GPU_SUPPORT = "HIP" ; then
+    PAC_CHECK_HEADER_LIB_OPTIONAL([rccl], [rccl.h], [rccl], [ncclCommInitAll])
```

Comment:
This should be `rcclCommInitAll` instead of `ncclCommInitAll`, I think.

Reply:
This line should be
`PAC_CHECK_HEADER_LIB_OPTIONAL([rccl], [rccl/rccl.h], [rccl], [ncclCommInitAll],, [-D__HIP_PLATFORM_AMD__])`.